[WIP] fix phone number

RonaldDijkstra opened the pull request:
Removes href=tel: and changes some other things to make sure the phone numbers are at least visible in any browser. 
 
Will come up with better solution, but for now this will do. 

#222

fatboypunk commented on source/localizable/contact/_contact.de.md.erb:
```diff
@@ -9,8 +9,8 @@ Zuiderpark 3
 
-T +49 (0)40 306988472
-F +31 (0)50 3186332
+T 040 306988472
```
why remove the landcode?

RonaldDijkstra replied:
Because I want to know if the rob's contact can see the phone number if we remove the landcode. We can't reproduce this problem ourselves

Op 2 jun. 2016 om 20:46 heeft Marcel Horlings <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


In source/localizable/contact/_contact.de.md.erb<https://github.com/DefactoSoftware/website/pull/225#discussion_r65596122>:

> @@ -9,8 +9,8 @@ Zuiderpark 3
>  9724 AD Groningen
>  **Niederlande**
>
> -T +49 (0)40 306988472
> -F +31 (0)50 3186332
> +T 040 306988472


why remove the landcode?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/DefactoSoftware/website/pull/225/files/9da8a581686728358e69ad6f72ba20d90c37bc03#r65596122>, or mute the thread<https://github.com/notifications/unsubscribe/APWuj8oB-QM9QGUEbkh9v6uW4UFSl0qJks5qHyTSgaJpZM4Isr2u>.

sn3p replied:
It looks like this is not part of the problem, so lets put them back?

fatboypunk commented on locales/de.yml:
```diff
@@ -66,8 +66,8 @@ de:
     heading2: "Wir lernen Sie gerne kennen, bei Ihnen, oder bei uns in Groningen."
-    phone: "040 306988472"
-    phone_link: "+4940306988472"
+    phone: <p>T&nbsp;&nbsp;040 306988472</p>
```
So why do we have the `&nbsp;` here? i see that's been done elsewhere to so i think @sn3p knows why?

fatboypunk commented:
<img width="177" alt="screen shot 2016-06-02 at 20 42 40" src="https://cloud.githubusercontent.com/assets/1089927/15756683/96e64660-2902-11e6-844d-94fdfbc25d19.png">
`T: `?
<img width="187" alt="screen shot 2016-06-02 at 20 43 27" src="https://cloud.githubusercontent.com/assets/1089927/15756695/a9eadae6-2902-11e6-9a04-a5372fece8b6.png">
I'd like the telephone number in the same color as the email

<img width="326" alt="screen shot 2016-06-02 at 20 44 20" src="https://cloud.githubusercontent.com/assets/1089927/15756721/cbbe2100-2902-11e6-8107-90db20e96823.png">
`T:` `F:`

RonaldDijkstra commented:
The color implies it's clickable, which I removed. Can change it though.

Op 2 jun. 2016 om 20:48 heeft Marcel Horlings <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


[screen shot 2016-06-02 at 20 42 40]<https://cloud.githubusercontent.com/assets/1089927/15756683/96e64660-2902-11e6-844d-94fdfbc25d19.png>
T:?
[screen shot 2016-06-02 at 20 43 27]<https://cloud.githubusercontent.com/assets/1089927/15756695/a9eadae6-2902-11e6-9a04-a5372fece8b6.png>
I'd like the telephone number in the same color as the email

[screen shot 2016-06-02 at 20 44 20]<https://cloud.githubusercontent.com/assets/1089927/15756721/cbbe2100-2902-11e6-8107-90db20e96823.png>
T: F:

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/DefactoSoftware/website/pull/225#issuecomment-223385756>, or mute the thread<https://github.com/notifications/unsubscribe/APWuj_VAJphAq1QIVJer-Y6JYU8P0VLFks5qHyVvgaJpZM4Isr2u>.

RonaldDijkstra commented:
- added the color in the footer
- added F: and T: 
- added the phone icon again
- we need to test if without the landcode the phone number displays correctly in the content of contact page

sn3p commented:
@RonaldDijkstra http://stackoverflow.com/questions/32005753/text-disappearing-in-ie-11-on-windows-7

> A bug in Windows 7 is causing text to disappear in IE10 and IE11 when the font-feature-settings css property is used. https://connect.microsoft.com/IE/feedbackdetail/view/831964

`font-feature-settings` mag er wat mij betreft af als dat het probleem op lost.

sn3p commented on locales/de.yml:
```diff
@@ -66,8 +66,8 @@ de:
     heading2: "Wir lernen Sie gerne kennen, bei Ihnen, oder bei uns in Groningen."
-    phone: "040 306988472"
-    phone_link: "+4940306988472"
+    phone: <span>040 306988472</span>
```
Do we need this `span`? Aside from that, adding html to translations is bad practice.

sn3p commented on locales/de.yml:
```diff
@@ -66,8 +66,8 @@ de:
-    phone: "040 306988472"
-    phone_link: "+4940306988472"
+    phone: <span>040 306988472</span>
+    phone_link: +4940306988472
```
This can be removed if we decide to remove the `href`. But lets try to [fix the links](https://github.com/DefactoSoftware/website/pull/225#issuecomment-223527265) first.

sn3p commented on locales/nl.yml:
```diff
@@ -53,8 +53,8 @@ nl:
-    phone: "050 3144832"
-    phone_link: "+31503144832"
+    phone: <span>050 3144832</span>
+    phone_link: +31503144832
```
See above.

RonaldDijkstra commented:
Thanks @sn3p
